bots/notify/src/main/java/org/openjdk/skara/bots/notify/PullRequestWorkItem.java

```diff
@@ -133,6 +135,9 @@ private String serializePrState(Collection<PullRequestState> added, Set<PullRequ
                                 } else {
                                     ret.putNull("state");
                                 }
+                                if (pr.targetBranch() != null) {
+                                    ret.put("targetBranch", JSON.of(pr.targetBranch()));
+                                }
```

Comment:
We have two options for handling non-replaced entries:
1: Do not add a null JSON for targetBranch
2: Add a null JSON for targetBranch

Currently, I chose option 1